Docker.DotNet/Endpoints/ContainerOperations.cs

```diff
@@ -387,25 +387,26 @@ public Task StartContainerExecAsync(string id, CancellationToken cancellationTok
 
         public async Task<MultiplexedStream> StartAndAttachContainerExecAsync(string id, bool tty, CancellationToken cancellationToken)
```

Comment:
We should probably add comments about why you would use one over the other now that we know why it hangs on input.